src/xtgeo/grid3d/_grid3d_utils.py

```diff
@@ -222,7 +222,7 @@ def _get_roff_type_and_size(
         return "float", value.size, value.size * (4 if is_binary else 5)
     if np.issubdtype(value.dtype, np.double):
         return "double", value.size, value.size * (8 if is_binary else 9)
-    if np.issubdtype(value.dtype, np.unicode_):
+    if np.issubdtype(value.dtype, np.str_):
```

Comment:
I will guess this is safe for numpy 1 also?

Reply:
Yep, I see it as far back as numpy 1.20 at least (may be earlier as well)